src/components/views/Login.tsx

```diff
@@ -15,14 +15,24 @@ specific components that belong to the main one in the same file.
  */
 const FormField = (props) => {
   return (
-    <div className="login field">
-      <label className="login label">{props.label}</label>
-      <input
-        className="login input"
-        placeholder="enter here.."
-        value={props.value}
-        onChange={(e) => props.onChange(e.target.value)}
-      />
+    <div className="register field">
+      <label className="register label">{props.label}</label>
+      {props.type === "password" ? (
+        <input
+          className="register input"
+          placeholder="enter here.."
+          type="password"
+          value={props.value}
+          onChange={(e) => props.onChange(e.target.value)}
+        />
+      ) : (
+        <input
+          className="register input"
+          placeholder="enter here.."
+          value={props.value}
+          onChange={(e) => props.onChange(e.target.value)}
+        />
+      )}
```

Comment:
code smell: duplicate code

Reply:
Not relevant for now: can be done later if time